app/forms/questionnaire_form.py

```diff
@@ -155,8 +155,9 @@ def validate_mutually_exclusive_question(
             question["validation"].get("messages") if "validation" in question else None
         )
         answers = (getattr(self, answer["id"]).data for answer in question["answers"])
-        is_only_checkboxes = all(
-            answer["type"] == "Checkbox" for answer in question["answers"]
+        is_only_checkboxes_or_radios = all(
+            answer["type"] == "Checkbox" or answer["type"] == "Radio"
```

Comment:
Did this need to change?. We can't have radios for the non-exclusive option but it only matters if it was checkbox anyway.